ui/src/app/(index)/_components/AfterLogin.tsx

```diff
@@ -0,0 +1,99 @@
+import User from '@/access/User';
+import Role from '@/access/Role';
+import Image from 'next/image';
+import {KeyRound} from 'lucide-react';
```

Comment:
For any import with {}, there is a common formatting convention to put one space inside the {}
For example:
```
import { KeyRound } from 'lucide-react';
```
Make sure to apply this to the rest of your PR.